### DISCUSSION TITLE
tech-detect found subdirs find vulnerabilities

### DISCUSSION THREAD

**Discussion by xshell888** [projectdiscovery/nuclei#3563] [Q&A] [answered]
### Please describe your feature request:

When scanning for vulnerabilities, tech-detect found that there were joomla or wordpress in subdirectories such as web or site or website, but when scanning, it found that it did not invoke related vulnerabilities to scan the secondary directory, only scanned /

### Describe the use case of this feature:

Some websites will put cms in a subdirectory, which also needs to be automatically scanned for vulnerabilities

**Comment by tarunKoyalwar** [comment, marked as answer]
@xshell888 , scanning subdirectories is supported by nuclei and this can be done while specifying input/target .
ex:
`nuclei -u https://scanme.sh/` => scans with / as base
`nuclei  -u https://scanme.sh/wordpress/` => scans with /wordpress as base 

what you are suggesting is runtime changes to input target . which doesn't seem like a great idea as nuclei is meant to be scriptable . If you have any ideas about  how we can implement this in templates (not in engine) pls do share . 

other than that I think it makes more sense to do this  at workflow/ automation level
